provisioning_client/CMakeLists.txt

```diff
@@ -81,9 +81,9 @@ elseif (${use_prov_client})
                 ./adapters/hsm_client_riot.c)
 
             # Include the riot directories
-            include_directories(./deps/RIoT/Emulator/DICE)
-            include_directories(./deps/RIoT/Emulator/RIoT)
-            include_directories(./deps/RIoT/Emulator/RIoT/RIoTCrypt/include)
+			include_directories(./deps/RIoT/Simulation/DICE)
+			include_directories(./deps/RIoT/Simulation/RIoT/Core)
+			include_directories(./deps/RIoT/Simulation/RIoT/Core/RIoTCrypt/include)
```

Comment:
ahhhhhh these CMake files make me cringe. I'm not a fan of how this is done here but in this case I think you can leave the `include_directories` instead of `target_include_directories`

Reply:
Fixed spacing